packages/core/src/ceramic.ts

```diff
@@ -701,12 +704,24 @@ export class Ceramic implements CeramicApi {
       chainId: this._anchorValidator.chainId,
     }
     const ipfsStatus = await this.dispatcher.ipfsNodeStatus()
+
+    let composeDB = undefined
+    if (!this._params.indexingConfig.disableComposedb) {
```

Comment:
I'd prefer us not save the `_params` - it's the job of the `IndexApi` to interpret the config so I think it would be better to ask questions of it than the config directly.

There isn't currently an `IndexApi.enabled` accessor, but you could easily add one that just checks if the underlying `databaseIndexApi` us defined or not.